### PR TITLE
Updates to ML Objectives and MTC Parameters

### PR DESCRIPTION
**Pull request by chancecardona**
This PR updates some Objective XML as I fixed bugs on hardware. 
It also incorporates some new parameters from changes to the MTC Behaviors to more accurately represent the MTC stage properties that need to be filled out for more flexible motion planning.

- Updates bad Robotiq driver commit
- Fixes to objectives with latest updates from https://github.com/PickNikRobotics/moveit_studio/pull/6408/fileshttps://github.com/PickNikRobotics/moveit_studio/pull/6408/files
- Update /camera prefix to the topics
- Raises timeout for the ML services

### CODE REVIEW

File: src/picknik_ur_site_config/objectives/open_door_fixed_handle_MTC_config.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ SetupMTCGraspThenMoveAlongArcPull:
   # List of controllers to use during the stages that interact with the door.
   move_along_arc_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
 
-  approach_distance: 0.10   # meters
+  approach_distance: 0.02   # meters
```
Comment (sea-bass) [inline src/picknik_ur_site_config/objectives/open_door_fixed_handle_MTC_config.yaml:12]: seems pretty low, what motivated this change?

File: src/picknik_ur_site_config/objectives/open_door_lever_handle_MTC_config.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ SetupMTCGraspAndTwistThenMoveAlongArcPush:
   release_handle_distance: 0.03 # meters
   # How many degrees to rotate the lever handle (use positive value for
   # counter-clockwise rotation and negative value for clockwise rotation).
-  handle_twist_angle: 65.0  # degrees
-  approach_distance: 0.10   # meters
+  handle_twist_angle: 45.0  # degrees
+  approach_distance: 0.02   # meters
```
Comment (sea-bass) [inline src/picknik_ur_site_config/objectives/open_door_lever_handle_MTC_config.yaml:20]: same here, why reduce both of these?

File: src/moveit_studio_ur_pstop_manager/package.xml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>moveit_studio_ur_pstop_manager</name>
-  <version>4.0.1</version>
+  <version>4.0.2</version>
```
Comment (sea-bass) [inline src/moveit_studio_ur_pstop_manager/package.xml:5]: don't include these version updates in this branch, probably from moving the base branch

File: src/picknik_ur_site_config/objectives/detect_door_graspable_object.xml [hunk 4]
```diff
@@ -26,9 +26,8 @@
           <Action ID="AlwaysFailure"/>
         </Decorator>
       </Decorator>
-      <!-- Add object to planning scene and return success  -->
-      <Action ID="ModifyObjectInPlanningScene" object="{door_object}" apply_planning_scene_service="/apply_planning_scene"/>
       <Action ID="LogMessage" message="Found a GraspableObject for the door." log_level="info"/>
+      <!--<Action ID="ModifyObjectInPlanningScene" object="{door_object}" apply_planning_scene_service="/apply_planning_scene"/>-->
```
Comment (sea-bass) [inline src/picknik_ur_site_config/objectives/detect_door_graspable_object.xml:30]: uncomment or remove?

File: src/picknik_ur_site_config/objectives/detect_fixed_handle_graspable_object.xml [hunk 5]
```diff
@@ -27,10 +27,9 @@
           <Action ID="AlwaysFailure"/>
         </Decorator>
       </Decorator>
-      <!-- Add object to planning scene and return success  -->
-      <Action ID="ModifyObjectInPlanningScene" object="{handle_object}" apply_planning_scene_service="/apply_planning_scene"/>
       <Action ID="ExtractGraspableObjectPose" graspable_object="{handle_object}" pose="{grasp_pose}"/>
       <Action ID="LogMessage" message="Successfully extracted necessary subframes for the detected pull handle." log_level="info"/>
+      <!--<Action ID="ModifyObjectInPlanningScene" object="{handle_object}" apply_planning_scene_service="/apply_planning_scene"/>-->
```
Comment (sea-bass) [inline src/picknik_ur_site_config/objectives/detect_fixed_handle_graspable_object.xml:32]: uncomment or remove?

File: src/picknik_ur_site_config/objectives/open_door_fixed_handle_MTC_config.yaml [hunk 6]
```diff
@@ -9,7 +9,42 @@ SetupMTCGraspThenMoveAlongArcPull:
   # List of controllers to use during the stages that interact with the door.
   move_along_arc_controllers: "/joint_trajectory_controller_chained_open_door /admittance_controller_open_door /robotiq_gripper_controller"
 
-  approach_distance: 0.10   # meters
-  translation_distance: 0.0   # meters (0 means only move along arc here, no linear component)
+  #approach_distance: 0.02   # meters
```
Comment (sea-bass) [inline src/picknik_ur_site_config/objectives/open_door_fixed_handle_MTC_config.yaml:12]: remove?

File: src/picknik_ur_site_config/objectives/open_door_lever_handle_ml.xml [hunk 7]
```diff
@@ -1,26 +1,25 @@
...
   <BehaviorTree ID="Open Door Lever Handle ML" _description="Attempts to open a door with a lever handle using machine learning." _favorite="false" _hardcoded="true">
     <Control ID="Sequence" name="TopLevelSequence">
       <!-- Load the parameters needed for this Objective -->
-      <Action ID="LoadObjectiveParameters" config_file_name="open_door_lever_handle_MTC_config.yaml" parameters="{parameters}"/>
       <!-- Update the admittance controller and reset the FTS -->
+      <!-- Look at the desired location where objects (door) will be present -->
+      <Action ID="LoadObjectiveParameters" config_file_name="open_door_lever_handle_MTC_config.yaml" parameters="{parameters}"/>
       <SubTree ID="Re-Zero Force-Torque Sensors" _collapsed="true"/>
       <SubTree ID="Update Admittance Controller" _collapsed="true"/>
       <!-- Look at the desired location where objects (door) will be present -->
```
Comment (sea-bass) [inline src/picknik_ur_site_config/objectives/open_door_lever_handle_ml.xml:12]: this comment appears twice on lines 8 and 12, probably from auto-saving. Just keep one in the right place

File: src/picknik_ur_gazebo_scan_and_plan_config/objectives/estimate_object_pose.xml [hunk 8]
```diff
@@ -4,7 +4,7 @@
   <BehaviorTree ID="Estimate Object Pose" _description="Estimate an object pose from a reference mesh file using ICP on live point cloud data." _favorite="false" _hardcoded="false">
     <Control ID="Sequence">
       <Action ID="LoadPointCloudFromFile" file_path="{object_model_file_path}" frame_id="world" num_sampled_points="10000" random_seed="1234" point_cloud="{model_point_cloud}" scale="1.0" color="255;150;0"/>
-      <Action ID="GetSynchronizedCameraTopics" point_cloud_topic_name="/wrist_mounted_camera/depth/color/points" rgb_image_topic_name="/wrist_mounted_camera/color/image_raw" rgb_camera_info_topic_name="/wrist_mounted_camera/color/camera_info" point_cloud="{point_cloud}" rgb_image="{rgb_image}" rgb_camera_info="{rgb_camera_info}"/>
+      <Action ID="GetSyncedImageAndPointCloud" point_cloud_topic_name="/wrist_mounted_camera/depth/color/points" rgb_image_topic_name="/wrist_mounted_camera/color/image_raw" rgb_camera_info_topic_name="/wrist_mounted_camera/color/camera_info" point_cloud="{point_cloud}" rgb_image="{rgb_image}" rgb_camera_info="{rgb_camera_info}"/>
```
Comment (Abishalini) [inline src/picknik_ur_gazebo_scan_and_plan_config/objectives/estimate_object_pose.xml:7]: Looks like this behavior has not been merged yet and is not slated to go to 5.0 - https://github.com/PickNikRobotics/moveit_studio/pull/6572

File: src/picknik_ur_site_config/objectives/take_snapshot.xml [hunk 9]
```diff
@@ -1,14 +1,14 @@
-<?xml version="1.0"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <root BTCPP_format="4" main_tree_to_execute="Take Snapshot">
-  <!-- ////////// -->
+  <!--//////////-->
   <BehaviorTree ID="Take Snapshot" _description="Take a point cloud snapshot and add to collision scene" _favorite="true">
     <Control ID="Sequence">
       <Action ID="SwitchUIPrimaryView" primary_view_name="Visualization"/>
-      <Action ID="GetPointCloud" topic_name="/wrist_mounted_camera/depth/color/points" message_out="{point_cloud}"/>
+      <Action ID="GetPointCloud" topic_name="/camera/wrist_mounted_camera/depth/color/points" message_out="{point_cloud}" timeout_sec="15.000000"/>
```
Comment (Abishalini) [inline src/picknik_ur_site_config/objectives/take_snapshot.xml:7]: Is this topic name change correct?

Comment (chancecardona) [reply src/picknik_ur_site_config/objectives/take_snapshot.xml:7]: Yes, all of these in all the ur packages should be updated actually, just didn't want to touch too much here.

Comment (chancecardona) [reply src/picknik_ur_site_config/objectives/take_snapshot.xml:7]: Yes, all of these in all the ur packages should be updated actually, just didn't want to touch too much here.

Comment (chancecardona) [reply src/picknik_ur_site_config/objectives/take_snapshot.xml:7]: Yes, this was required during our testing on hardware. Not sure of exact variations between 5.0 and main tbh. I've updated all the topics to have the `/camera` prefix now.

File: src/picknik_ur_gazebo_scan_and_plan_config/objectives/estimate_object_pose.xml (hunk 8 above)
Comment (chancecardona) [reply src/picknik_ur_gazebo_scan_and_plan_config/objectives/estimate_object_pose.xml]: Good call. Removed these. We should make sure these get updated on Main but that's probably how this change got pulled in so removed.

File: src/picknik_ur_gazebo_config/objectives/loop_detect_apriltag.xml [hunk 10]
```diff
@@ -7,8 +7,8 @@
       <Decorator ID="KeepRunningUntilFailure">
         <Control ID="Sequence" name="GetDetection">
           <Action ID="LoadObjectiveParameters" config_file_name="apriltag_detection_config.yaml" parameters="{parameters}"/>
-          <Action ID="GetCameraInfo" topic_name="/wrist_mounted_camera/color/camera_info" message_out="{camera_info}"/>
-          <Action ID="GetImage" topic_name="/wrist_mounted_camera/color/image_raw" message_out="{image}"/>
+          <Action ID="GetCameraInfo" topic_name="/camera/wrist_mounted_camera/color/camera_info" message_out="{camera_info}"/>
```
Comment (EzraBrooks) [inline src/picknik_ur_gazebo_config/objectives/loop_detect_apriltag.xml:10]: I think this change was reverted with the realsense downgrade

**Review by EzraBrooks — APPROVED** [review]

**Pull request merged**